File: src/main/java/com/koteldlc/client/module/modules/visual/TargetESP.java
package com.koteldlc.client.module.modules.visual;

public class TargetESP extends BaseEspModule {
    public TargetESP() {
        super("TargetESP", "Яркая подсветка текущей цели KillAura/AimAssist", 0xFFFF00FF);
        blink.setValue(true);
        thickness.setValue(3.5);
    }
}

File: kotlin-mod/src/main/kotlin/com/koteldlc/client/modules/visual/TargetESP.kt
package com.koteldlc.client.modules.visual

class TargetESP : BaseEspModule("TargetESP") {
    init {
        settings["color"] = 0xFFFF00FF.toInt()
        settings["thickness"] = 3.5
        settings["blink"] = true
    }

    override fun onRenderEsp() {
        // Подсветка текущей цели KillAura.
    }
}
